client/app/billing/autoRenew/actions/terminateHostingWeb/hosting-web.component.js

```diff
@@ -0,0 +1,11 @@
+import controller from './billing-autoRenew-terminateHostingWeb.controller';
```

Comment:
Looks like the `terminate` modals could be refactored into one component, couldn't them ?

Reply:
Could have a look to do so, indeed